index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
```

Comment:
it seemed to be empty and I couldn't see anything that would be making it.
Maybe it should have been a different PR